src/components/Analysis/Applications/CreateApplicationDialog.vue

```diff
@@ -210,6 +240,36 @@ export default {
           label: "Postprocessor",
         },
       ],
+      cpuItems:[
+        {
+          value: 1024,
+          label: "1 CPU",
+        },
+        {
+          value: 2048,
+          label: "2 CPU",
+        },
+        {
+          value: 4096,
+          label: "4 CPU",
+        },
+        {
+          value: 8192,
+          label: "8 CPU",
+        }
```

Comment:
seems like 16 vCPU is missing from the list, not sure if that needs to be excluded.